GitUpKit/Views/GIQuickViewController.m

```diff
@@ -235,3 +250,212 @@ - (BOOL)diffFilesViewController:(GIDiffFilesViewController*)controller handleKey
+                               ];
+      [NSLayoutConstraint activateConstraints:constraints];
+    }
+    
+    [self toggleLeftView];
+  } else {
+    NSView *leftView = self.commitListViewController.view;
+    if (leftView.superview != nil) {
+      NSView *superview = leftView.superview;
+      NSArray *constraints = @[
+        [NSLayoutConstraint constraintWithItem:leftView attribute:NSLayoutAttributeLeft relatedBy:NSLayoutRelationEqual toItem:superview attribute:NSLayoutAttributeLeft multiplier:1.0 constant:0.0],
+        [NSLayoutConstraint constraintWithItem:leftView attribute:NSLayoutAttributeWidth relatedBy:NSLayoutRelationEqual toItem:superview attribute:NSLayoutAttributeWidth multiplier:0.3 constant:0.0]
+      ];
+      NSArray *verticalConstraints = [NSLayoutConstraint constraintsWithVisualFormat:@"V:|[view]|" options:0 metrics:nil views:@{@"view": leftView}];
+      [NSLayoutConstraint activateConstraints:[constraints arrayByAddingObjectsFromArray:verticalConstraints]];
+    }
+    NSView *rightView = self.quickViewController.view;
+    if (rightView.superview != nil) {
+      NSView *superview = rightView.superview;
+      self.hiddenConstraint = [NSLayoutConstraint constraintWithItem:rightView attribute:NSLayoutAttributeLeft relatedBy:NSLayoutRelationEqual toItem:superview attribute:NSLayoutAttributeLeft multiplier:1.0 constant:0.0];
+      self.revealedConstraint = [NSLayoutConstraint constraintWithItem:rightView attribute:NSLayoutAttributeLeft relatedBy:NSLayoutRelationEqual toItem:leftView attribute:NSLayoutAttributeRight multiplier:1.0 constant:0.0];
+      NSArray *constraints = @[
+        [NSLayoutConstraint constraintWithItem:rightView attribute:NSLayoutAttributeRight relatedBy:NSLayoutRelationEqual toItem:superview attribute:NSLayoutAttributeRight multiplier:1.0 constant:0.0],
+      ];
+      NSArray *verticalConstraints = [NSLayoutConstraint constraintsWithVisualFormat:@"V:|[view]|" options:0 metrics:nil views:@{@"view": rightView}];
+      [NSLayoutConstraint activateConstraints:[constraints arrayByAddingObjectsFromArray:verticalConstraints]];
+    }
+    
+    [self toggleLeftView];
+  }
+}
+
+#pragma mark - View Lifecycle
+- (void)loadView {
+  self.view = [[GIView alloc] initWithFrame:NSScreen.mainScreen.frame];
+}
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+  NSViewController *leftController = ({
+    GICommitListViewController *commitsList = [[GICommitListViewController alloc] initWithRepository:self.repository];
+    // setup?
+    commitsList.delegate = self;
+    commitsList;
+  });
+  NSViewController *rightController = ({
+    GIQuickViewController *quickView = [[GIQuickViewController alloc] initWithRepository:self.repository];
+    // setup?
+    __weak typeof(self) weakSelf = self;
+    quickView.willShowContextualMenu = ^(NSMenu *menu, GCDiffDelta *delta, GCIndexConflict *conflict) {
+      [weakSelf willShowContextualMenu:menu delta:delta conflict:conflict];
+    };
+    quickView;
+  });
```

Comment:
Just construct the properties directly and don't rely on `addChilrdViewController:` to represent which is which. Simplify the logic and simply initialize the property and set the delegate. This block assigning thing is very bizarre.